README.md

```diff
@@ -1,4 +1,6 @@
-[![Nightly CI status master](https://ci.riot-os.org/RIOT-OS/RIOT/master/latest/badge.svg)](https://ci.riot-os.org/RIOT-OS/RIOT/master/latest/output.html)
+[![Nightly CI status master][ci-badge]](https://ci.riot-os.org/RIOT-OS/RIOT/master/latest/output.html)
```

Comment:
I would rather go for `master-ci-badge`. I think we should be prepared for a future where we might want to nightly build other branches than master. Also: Why not also move the link down with `master-ci-dump` or something?